.github/ISSUE_TEMPLATE.md

```diff
@@ -13,7 +13,7 @@ the problem ourselves.
 ### Environment and Platform
 
 - OS: Linux, Windows or BSD / OS X?
-- Python: 2.6, 2.7, 3.5, PyPy?
+- Python: 2.7, 3.5, PyPy?
```

Comment:
I think the whole thing is/was ment as question since it is part of the issue template -- but yeah, I'll remove the question marks (also for the OS above)